### ISSUE TITLE
Alternative path to closurizing constructors?

### ISSUE THREAD

**Issue by alan-knight** [gbracha/metaclasses#1]
This seems like it would provide a way of closurizing a constructor, without needing the extra syntax in that proposal (https://github.com/gbracha/generalizedTearOffs)  Instead of MyClass#constructorName you could write MyClass.constructorName. It doesn't address operators or getters/setters, but it seems like there's some duplication there.

**Comment by gbracha** [comment]
Yes, I know.  This was mentioned in my first draft (which predated the DEP process).  After discussion, I was asked to go ahead with the tear-off proposal independently.
